pages/about.html

```diff
@@ -100,28 +100,79 @@
         </ol>
       </nav>
```

Comment:
- [ ] A user should be able to navigate between the home and about pages but that is not the case now. Kindly rectify this issue so that when a user clicks the about link on the home page, the user is taken to the about page and vice versa. This should also be applied to the mobile menu 💯 
- [ ] Also, the about page should have a dark navbar. Kindly add this too 👍

Reply:
Fixed